stdlib/multiprocessing/forkserver.pyi

```diff
@@ -0,0 +1,44 @@
+from _typeshed import FileDescriptorLike, StrOrBytesPath
+from collections.abc import Iterable, Mapping, Sequence
+from signal import _HANDLER, _SIGNUM
+from struct import Struct
+from threading import Lock
+from typing import Any
+
+__all__ = ["ensure_running", "get_inherited_fds", "connect_to_new_process", "set_forkserver_preload"]
+
+MAXFDS_TO_SEND: int = ...
+SIGNED_STRUCT: Struct = ...
+
+class ForkServer:
+    _forkserver_address: StrOrBytesPath | None
+    _forkserver_alive_fd: int | None
+    _forkserver_pid: int | None
+    _inherited_fds: Sequence[int] | None
+    _lock: Lock
+    _preload_modules: list[str]
+
+    def __init__(self) -> None: ...
+    def _stop(self) -> None: ...
+    def _stop_unlocked(self) -> None: ...
+    def set_forkserver_preload(self, modules_names: list[str]) -> None: ...
+    def get_inherited_fds(self) -> Sequence[int] | None: ...
```

Comment:
For stubs, be as permissive as possible for argument annotations, and as precise as possible for return annotations. The docstring says this returns a "list of fds", and "list" is more precise than "Sequence".

```suggestion
    def get_inherited_fds(self) -> list[int] | None: ...
```